CalcualtorLogic.js

```diff
@@ -171,14 +171,15 @@ class Calculator {
         this.updateResultElement();
     }
     evaluate() {
-        if (this.lastValue && this.currentResult !== null) {
+        if (this.lastValue && this.currentResult !== null && this.operator !== null) {
```

Comment:
You're checking if `operator` is available (not null)
but you're not resetting the operator flag to null 
I would add 
`this.opeator = null `

Reply:
I do not know how the equality changed to ===

I set it to = 
And checked the calculator 
When it is ===
The problem is same problem

Reply:
is this comment solved ?

Reply:
Yes it is 
So can I merge and close and checkout to master and pull

Thanks